src/plugins/lua/metadata_exporter.lua

```diff
@@ -151,6 +153,18 @@ local function get_general_metadata(task, flatten, no_content)
       return 'unknown'
     end
   end
+
+  local scan_real = task:get_scan_time()
+  scan_real = math.floor(scan_real * 1000)
+  if scan_real < 0 then
+    rspamd_logger.messagex(task,
```

Comment:
Does it really happen? Just curious :)

Reply:
Ideally it shouldn't, though it's in fact a copy of https://github.com/rspamd/rspamd/commit/6e105da2247e85be60ff31f6046dc8e24c87d807 - when implemented I'm guessing there's a reason why it was done.

I can remove the condition, that's no problem - I'd simply keep it consistent with the elastic plugin

Reply:
Keeping it consistent is a good thing, no worries. Thank you!